<?xml version="1.0"?>
<launch>
	<node pkg="weiss_kms40" type="Kms40Driver" name="kms40" output="screen" cwd="node">
		<rosparam command="load" file="$(find weiss_kms40)/launch/config.yaml"/>
	</node>
	<node pkg="rosservice" type="rosservice" name="force_sensor_tare" args="call --wait /kms40/tare true"/>

	<!--
	<arg name="device_name"                default="/dev/ttyUSB0"/>
	<param name="device_name"              value="$(arg device_name)"/>

	<arg name="baud_rate"                  default="1000000"/>
	<param name="baud_rate"                value="$(arg baud_rate)"/>

	<arg name="scan_range"                 default="10"/>
  <param name="scan_range"               value="$(arg scan_range)"/>

	<arg name="profile_velocity"           default="200"/>
	<param name="profile_velocity"         value="$(arg profile_velocity)"/>

	<arg name="profile_acceleration"       default="50"/>
  <param name="profile_acceleration"     value="$(arg profile_acceleration)"/>

  <node name="position_control" pkg="dynamixel_workbench_controllers" type="position_control" required="true" output="screen"/>
	<node pkg="weiss_kms40" type="Kms40Driver" name="kms40" output="screen" cwd="node">
		<rosparam command="load" file="$(find weiss_kms40)/launch/config.yaml"/>
	</node>
	<node name="sensor_data" pkg="weiss_kms40" type="sensor_data" required="true" output="screen"/>

	<node pkg="rosservice" type="rosservice" name="force_sensor_tare" args="call --wait /kms40/tare true"/>

  <node pkg="rosservice" type="rosservice" name="dyn1" args="call --wait /joint_command -- raw 1 110"/>
	<node pkg="rosservice" type="rosservice" name="dyn2" args="call --wait /joint_command -- raw 2 370"/>
-->
</launch>
 <!--DYN10 - 115-->
 <!--DYN20 - 370-->
